### ISSUE TITLE
Read and Write comments

### ISSUE THREAD

**Issue by Linuus** [joker1007/unite-pull-request#7]
Is it possible to read and write comments for the current PR?

I looked in the code and saw some stuff regarding commenting it seems, but I can't find anything in the Readme.

I'm looking for something which have about the same features as:
https://github.com/codegram/vim-codereview

which seems to not be very well maintained anymore.

**Comment by joker1007** [comment]
Yes, unite-pull-request can write comments, but cannot read comments.
I don't write description about this feature, because it was experimental.
I think that it is tested enough, I will update doc and README soon.

about read comments feature...
Please wait. :bow: 

Thanks!! :sparkles:

**Comment by alexandre** [comment]
Hello, @joker1007.

Thank you for sharing your work. I would like to know if it still a possible feature.
